test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -18,6 +18,26 @@
     "sagemaker",
 ]
 
+COMMON_PYTORCH_TRAINING_UTILITY_PACKAGES_IMPORT = [
+    "torch",
+    "torchvision",
+    "torchtext",
+    "torchaudio",
```

Comment:
While it's not bad to have these checks, these are unnecessary as we have integration tests that specifically test the functionality of these packages anyway.